demo/gpu-test4.yaml

```diff
@@ -38,7 +35,28 @@ spec:
     args: ["export; sleep 9999"]
     resources:
       claims:
-      - name: gpus
+      - name: shared-gpu
   resourceClaims:
-  - name: gpus
-    resourceClaimTemplateName: multiple-gpus
+  - name: shared-gpu
+    resourceClaimName: single-gpu
+
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  namespace: gpu-test4
+  name: pod1
+  labels:
+    app: pod
+spec:
+  containers:
+  - name: ctr0
+    image: ubuntu:22.04
+    command: ["bash", "-c"]
+    args: ["export; sleep 9999"]
```

Comment:
minor: Is there a pause container we could use instead?

Reply:
Maybe -- we can look into it as a follow-up.

Reply:
Yes, was more of a passing note.